.github/workflows/ci.yml

```diff
@@ -352,3 +352,31 @@ jobs:
       - name: build Deno
         run: |
           cargo clippy --manifest-path cts_runner/Cargo.toml
+
+  cargo-deny-check-advisories:
+    name: "Run `cargo deny check advisories`"
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v3
+
+      - name: Run `cargo deny check`
+        uses: ErichDonGubler/cargo-deny-action@update-deny
+        with:
+          command: check advisories
+          arguments: --all-features --workspace
+          rust-version: ${{ env.RUST_VERSION }}
+
+  cargo-deny-check-rest:
+    name: "Run `cargo deny check` for everything else"
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v3
+
+      - name: Run `cargo deny check`
+        uses: ErichDonGubler/cargo-deny-action@update-deny
```

Comment:
What's wrong/different about the first-party https://github.com/EmbarkStudios/cargo-deny-action